### PR TITLE
test=develop, fix space_to_depth_doc

### PR DESCRIPTION
**Pull request by JiabinYang**
fix space_to_depth example by executable code block
fx https://github.com/PaddlePaddle/Paddle/issues/16267

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -9654,9 +9654,16 @@ def space_to_depth(x, blocksize, name=None):
...
 
             data = fluid.layers.data(
-                name='data', shape=[1, 4, 2, 2], dtype='float32')
+                name='data', shape=[1, 4, 2, 2], dtype='float32', append_batch_size=False)
             space_to_depthed = fluid.layers.space_to_depth(
                 x=data, blocksize=2)
+
+            exe = fluid.Executor(fluid.CUDAPlace(0))
+            data_np = np.arange(0,16).reshape((1,4,2,2)).astype('float32')
+            out = exe.run(fluid.default_startup_program())
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py]: `space_to_depth` doesn't have a parameter, so line9663 is unnecessary.

Comment (JiabinYang) [reply python/paddle/fluid/layers/nn.py]: done, thx

**Review by chengduoZH — DISMISSED** [review]

**Review by chengduoZH — APPROVED** [review]

**Comment by shanyi15** [comment]
image

**Review by shanyi15 — APPROVED** [review]
LG

**Pull request merged**